content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,234 @@
+---
+title: Using the SDKs
+section: asset-tracking
+index: 20
+meta_description: "Ably Asset Tracking SDKs provide an easy way to track multiple assets with realtime location updates powered by the Ably realtime network and Mapbox Navigation SDK with location enhancement."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Ably realtime, map matching, location tracking, Publishing SDK, Subscribing SDK"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Supported platforms#supported-platforms
+    - SDK Repositories#repositories
+    - Prerequisites#prerequisites
+    - Installing the SDK#installing-sdk
+    - Using the Publishing SDK#publishing-sdk
+    - Using the Subscribing SDK#subscribing-sdk
+    - See also#see-also
+---
+
+This topic explains how to get started using the Ably Asset Tracking SDKs.
+
+h2(#supported-platforms). Supported platforms
+
+There are two Asset Tracking SDKs, one for publishing and one for subscribing. The following platforms are supported:
+
+* Android (Java and Kotlin) - publisher and subscriber SDK
+* iOS (Objective-C and Swift) - publisher and subscriber SDK
+* Web (JavaScript) - subscriber SDK
```

Comment:
I wonder if it's worth flaunting the fact that the JavaScript SDK has first class TypeScript support? @kavalerov WDYT?

Reply:
Seems worth mentioning. Added note.